cmake/modules/FindDAOS.cmake

```diff
@@ -32,6 +32,9 @@ set(DAOS_INCLUDE_DIRS ${DAOS_INCLUDE_DIR} ${DAOS_FS_INCLUDE_DIR})
 set(DAOS_LIBRARIES ${DAOS_LIBRARY} ${DAOS_FS_LIBRARY} ${DAOS_UNS_LIBRARY})
 
 include(FindPackageHandleStandardArgs)
+include_directories( ${PC_DAOS_INCLUDEDIR} )
+link_directories( ${PC_DAOS_LIBDIR} )
+
```

Comment:
Are these necessary here?

Reply:
yes, for when building DAOS without forcing the folder to build in.  These are for locating the files